build.gradle.kts

```diff
@@ -1,223 +1,236 @@
-
 import edu.wpi.first.wpilib.versioning.ReleaseType
 import org.gradle.api.Project
-import org.gradle.api.plugins.quality.FindBugs
```

Comment:
Do not remove.  Replace with spot bugs.  See #74

Reply:
Fixed